### ISSUE TITLE
Create/Register CDX Dev accounts

### ISSUE THREAD

**Issue by JorjaComer** [US-EPA-CAMD/PI-1#1278]
**CDX DEV URL** - https://dev.epacdx.net/
- Create/Register CDX Dev accounts for David, Joshua, Jason
- Jason will register for the Designated Representative to be able to approve dev and test accounts
- Joshua and David will register for the Data Prepare Agent

**REGISTRATION STEPS**

1. Search for and select the following Program
![image](https://user-images.githubusercontent.com/66504934/116914214-0e0b2e00-ac10-11eb-81fe-b1699f867c1a.png)

2. Select the Data Prepare Agent
![image](https://user-images.githubusercontent.com/66504934/116914346-3b57dc00-ac10-11eb-928d-a5444b633c09.png)

3. Fill out personal info in Part 1
    - userid use the 1st letter of first name and full last name all lowercase followed by "-dp"
    - example: jwhitehead-dp
    - for security questions select questions and answers that you will remember

4. Part #2 search for epa and select the following organization
![image](https://user-images.githubusercontent.com/66504934/116914644-98539200-ac10-11eb-9706-053ec4ea0241.png)

5. Use your CVP email address as you will get a verification code to use in the next step
![image](https://user-images.githubusercontent.com/66504934/116914780-cfc23e80-ac10-11eb-8121-18c8d3a49d9d.png)

6. Enter the code sent to your email and click create account
![image](https://user-images.githubusercontent.com/66504934/116915890-3dbb3580-ac12-11eb-8e1e-ba440e730979.png)

7. Enter Role Sponsorship email (jasonwhitehead@cvpcorp.com) and click submit
![image](https://user-images.githubusercontent.com/66504934/116916045-6c391080-ac12-11eb-8242-d83d35a3ca02.png)

8. Review Role Sponsorship email and make sure to check the Yes box
![image](https://user-images.githubusercontent.com/66504934/116916227-ab676180-ac12-11eb-9084-98a443c3c67c.png)

9. Confirm Role Sponsorship email
![image](https://user-images.githubusercontent.com/66504934/116916527-1022bc00-ac13-11eb-95ac-e5f6f0f5a9e8.png)

10. Role Sponsorship should get an email, however, let them know so that they can approve the request. Once approved you should be able to login.
![image](https://user-images.githubusercontent.com/66504934/116916825-714a8f80-ac13-11eb-86f3-c0ebb9384e1b.png)

**Comment by jwhitehead77** [comment]
Joshua and I have already done this so just waiting on David to create his account and me to approve to close

**Comment by david-cvp** [comment]
5/4 Mike approved Joshua and I's accounts. No longer awaiting ESA, now accounts are active
